app/2.0/docs/tools/create-app-polymer-cli.md

```diff
@@ -0,0 +1,124 @@
+
+    Where <code><var>app</var></code> is the name of your project directory.
+
+1.  Initialize your app. Polymer CLI asks you a few questions
+    as it sets up your app.
+
+        polymer init
+
+1.  Select `polymer-2-application`.
+
+1.  Enter a name for your app. Defaults to the name of the current directory.
+
+1.  Enter a name for the main element in your project. The main element is the
+    top-most, application-level element of your app. Defaults to the name of
+    the current directory, followed by `-app`.
+
+    The code samples throughout this doc use the example app element name
+    <code><var>my-app</var></code>. When creating your app you'll want to
+    replace any instance of <code><var>my-app</var></code> with the name of
+    your main element.
+    {.alert .alert-info}
+
+1.  Enter a description for your app.
+
+The Polymer CLI generates your app and installs its dependencies.
+
+## App project layout
+
+After creating your app, Polymer CLI generates the following files and directories:
+
+*   `bower.json`. Configuration file for Bower.
+*   `bower_components/`. Project dependencies. See [Manage dependencies](#dependencies).
+*   `index.html`. Entrypoint page of the app.
+*   `src/`<code><var>my-app</var></code>/<code><var>my-app</var></code>`.html`.
+    Source code for main element.
+*   `test/`<code><var>my-app</var></code>/<code><var>my-app</var></code>`_test.html`. Tests 
+    for main element.
+
+### Add elements
+
+You may want to compose your main element out of smaller elements specific to your app. These 
+application-specific elements should be defined in the `src` directory, at the same level as 
+<code><var>my-app</var></code>.
+
+<code>
+    app/
+      src/
+        <var>my-app</var>/
+          <var>my-app</var>.html
+        <var>my-el</var>/
+          <var>my-el</var>.html
+</code>
+
+To add another element to the project:
+
+1. Create a new folder under src.
+   
+<code>
+   mkdir src/<var>my-el</var>
+</code>
+   
+2. Create an HTML import for the new element. You can use the existing app element as a starting point.
+   
+3. To use the new element, you'll need to import it into your application element (for example, <code><var>my-app</var>.html</code>) with an "import" link:
+   
+<code>
+   <link rel="import" href="/src/<var>my-el</var>/<var>my-el</var>.html">
+</code>
+   
+   Don't use `polymer init` to create an element project inside your app project.
+
+## Manage dependencies in an application project {#dependencies}
+
+Applications should use relative URLs to import other source files and dependencies. But because 
+applications are served independently, they can properly reach into the dependencies directory for 
+dependencies.
+
+    <!-- from a 'src/some-application.html' file in your application -->
+    <link rel="import" href="../bower_components/polymer/polymer.html">
+
+Only one file should use absolute URLs for imports: your "entrypoint" (default: "index.html"). 
```

Comment:
I think we can delete these two paragraphs (this and the following). Although the current basic app template uses absolute URLs, PSK now uses relative URLs in the entrypoint and I think we'll be going that way in the future. So we probably don't need to call this out.